modules/features2d/src/kaze/AKAZEFeatures.cpp

```diff
@@ -398,7 +398,7 @@ compute_diffusivity(InputArray Lx, InputArray Ly, OutputArray Lflow, float kcont
       charbonnier_diffusivity(Lx, Ly, Lflow, kcontrast);
     break;
     default:
-      CV_Error(diffusivity, "Diffusivity is not supported");
+      CV_Error(static_cast<int>(diffusivity), "Diffusivity is not supported");
```

Comment:
Initial code is buggy. The first argument for `CV_Error()` should be from `cv::Error` enum:
```
CV_Error_(Error::StsError, ("Diffusivity is not supported: %d", static_cast<int>(diffusivity)));
```